src/reducerFn.js

```diff
@@ -14,21 +14,20 @@ export default function reducerFn(initialState, actions={}, reducer) {
   const { actionFetch, actionSuccess, actionFail,
     actionReset, actionCache, actionAbort } = actions;
   return (state=initialState, action)=> {
+    const params = action.params || {};
```

Comment:
Params are stored in action.request.params. This is currently always defaulting to {}